typescript/infra/config/environments/test/index.ts

```diff
@@ -1,8 +1,17 @@
-export { core } from './core';
-export { governance } from './governance';
-export { agent } from './agent';
-export {
-  domainNames,
-  registerMultiProvider,
-  registerMultiProviderTest,
-} from './domains';
+import { CoreEnvironmentConfig } from '../../../src/config';
+import { configs } from '../../networks/testnets';
+import { core } from './core';
+import { governance } from './governance';
+import { agent } from './agent';
+import { metrics } from './metrics';
+import { infra } from './infra';
+
+export const environment: CoreEnvironmentConfig = {
+  domains: ['alfajores', 'kovan', 'mumbai', 'fuji'],
```

Comment:
should these be a shared higher level `testnetDomains` type somewhere? feels a little buried here

Reply:
Check out `networks.ts`